### ISSUE TITLE
Accessibility – Closed items should have `aria-hidden="true"` attributes set

### ISSUE THREAD

**Issue by kpeatt** [mobify/bellows#24]
Bellows should automatically walk through any item that doesn't have a `.bellows--is-open` class and attach `aria-hidden="true"` attributes to `.bellows__content-wrapper`. On open or close, this attribute should be removed or added as necessary.
